flow/designs/gf12/ariane/config_hier.mk

```diff
@@ -0,0 +1,40 @@
+export DESIGN_NAME = ariane
+export PLATFORM    = gf12
+
+#export FLOW_VARIANT = hier
+export FLOW_VARIANT = hier_rtlmp
+export SYNTH_HIERARCHICAL = 1
+export MAX_UNGROUP_SIZE = 100
+export RTLMP_FLOW = True
+
+export VERILOG_FILES = ./designs/src/$(DESIGN_NAME)/ariane.sv2v.v \
+						./designs/$(PLATFORM)/$(DESIGN_NAME)/macros.v
+
+export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NAME)/constraint.sdc
+
+export WRAP_LEFS = $(PLATFORM_DIR)/lef/gf12lp_1rf_lg8_w64_byte.lef
+
+export WRAP_LIBS = $(PLATFORM_DIR)/lib/gf12lp_1rf_lg8_w64_byte_sspg_sigcmax_0p72v_0p72v_125c.lib
+
+export ADDITIONAL_GDS  = $(PLATFORM_DIR)/gds/gf12lp_1rf_lg8_w64_byte.gds2
```

Comment:
I would include config.mk and rather than repeat all these common settings.  Here and in the other designs.

Reply:
Yes, if the settings are rules rather than exceptions, then it should go in the platform config.

Reply:
Pushed changes to for config_hier.mk to  include config.mk and set only the new values in config_hier.mk
Also fixed the issue about using tabs instead of spaces in some of the config files
Changed '?=' to '=' for PLACE_DENSITY setting in the files